requirements.txt

```diff
@@ -3,3 +3,4 @@ PyHive[hive]>=0.6.0,<0.7.0
 pyodbc>=4.0.30
 sqlparams>=3.0.0
 thrift>=0.11.0,<0.12.0
+pure-transport>=0.2.0
```

Comment:
Could we put an upper bound on the version here, since it's version-0 software?

You'll need to add this to the extra setup requirements as well:
https://github.com/fishtown-analytics/dbt-spark/blob/dff1b613ddf87e4e72e8a47475bcfd1d55796a5c/setup.py#L41-L44

I'm inclined to bundle it in with the larger `PyHive` extra, unless there's a good reason to bundle it separately.